### DISCUSSION TITLE
Integrity error on new database

### DISCUSSION THREAD

**Discussion by enzedonline** [wagtail/wagtail-localize#388] [Q&A] [answered]
Trying to get my production server filled quite urgently. Vanilla db, created the contact page, tried to translate and get an integrity error telling me key/locale already exists.
Deleted the page and tried again, can't even create the new page now.
Any help really appreciated, this is really urgent for me.... I don't even know where/how to start troubleshooting this in the db.

Exception Type: | IntegrityError
-- | --
duplicate key value violates unique constraint "contact_formfield_translation_key_locale_id_b8bc5c6e_uniq" DETAIL:  Key (translation_key, locale_id)=(8f8fd9b5-1dc9-4b85-aad7-84abbe678a59, 3) already exists.

**Comment by enzedonline** [comment, marked as answer]
This might be a bug with wagtail-localize. It only affected the contact page model (based on WagtailCaptchaEmailForm). 

I got around it by disabling the automatic sync in the locale settings before doing the initial translation. Once I'd done that, I could re-enable the automatic sync. It's the only page on the site with form fields and the only one affected by this issue. 

I haven't tried this on a test site yet. Maybe it affects other types of forms also.

**Reply by kaedroho** [reply]
Sorry for the delay in responding, would you mind opening an issue for this and sharing your model?

**Reply by enzedonline** [reply]
Sure, will do. Maybe related to #346?
